TESTING.md

```
@@ -6,8 +6,7 @@ This is the single source of truth for Vera's testing infrastructure, coverage d
 
 | Metric | Value |
 |--------|-------|
-| **Tests** | 12,141 across 176 files (~157,000 lines of test code; 11,942 passed + 26 stress-deselected, 173 skipped) |
-| **Compiler code coverage** | 95% Python, 87% JavaScript (CI minimum: 80%) |
+| **Tests** | 12,143 across 177 files (~157,000 lines of test code; 11,944 passed + 26 stress-deselected, 173 skipped) || **Compiler code coverage** | 95% Python, 87% JavaScript (CI minimum: 80%) |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 2; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)